src/shared/Atlassian.Bitbucket/BitbucketConstants.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+using System;
+
+namespace Atlassian.Bitbucket
+{
+    public static class BitbucketConstants
+    {
+        public const string BitbucketBaseUrlHost = "bitbucket.org";
+        public static readonly Uri BitbucketApiUri = new Uri("https://api.bitbucket.org");
+        public const string BitbucketAuthHelperName = "Atlassian.Bitbucket.UI";
+
+        public const string OAuth2ClientId = "b5AKdPfpgFdEGpKzPE";
```

Comment:
This is a new application "Git Credential Manager" still 'owned' by myself. We can swap this out for the app ID used by the old GCM, which was the same as Sourcetree's! I'd like to try and reach out and whitelist our application (and make it more Microsoft 'official').

Reply:
and this is registered through BitBucket?

Reply:
Yes. This is registered through Bitbucket. I've started the process of transferring the GitHub GCM app to a verified organisation (following MS policy), so I'd need to do the same for Bitbucket, but that's new ground AFAIK. Perhaps the Azure Pipelines folks know more here?
The alternative is to continue to 'steal' Sourcetree's ID?

Reply:
I'll continue to use Sourcetree's values until we can do this.